### PR TITLE
Map old env variables to new ones

### LINKED ISSUE #179: Map services variables

### LINKED ISSUE #180: Change DAPP_DEPLOY_BLOCK_HASH to DAPP_DEPLOY_BLOCK_NUMBER
## 📚 Context

We use the dapp deployment block hash in the dispatcher config. However, when deploying the DApp contract locally, a new block hash is generated, making it difficult to configure the node.

## ✔️ Solution

Use the block number instead of the block hash to make it easier to configure the node.

### PR DESCRIPTION
**Pull request by gligneul**
Close #180 
Close #179

### CODE REVIEW

File: cmd/cartesi-rollups-node/services.go [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+		panic("impossible")
+	}
+}
+
+func newAdvanceRunner() services.Service {
+	var s services.Service
+	s.Name = "advance-runner"
+	s.HealthcheckPort = advanceRunnerPort
+	s.Path = "cartesi-rollups-advance-runner"
+	s.Env = append(s.Env, "LOG_ENABLE_TIMESTAMP=false")
```
Comment (renan061) [inline cmd/cartesi-rollups-node/services.go:55]: Aren't these configurable?

**Review by renan061 — APPROVED** [review]
Noice

File: build/node-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      CARTESI_BLOCKCHAIN_HTTP_ENDPOINT: "http://devnet:8545"
+      CARTESI_BLOCKCHAIN_WS_ENDPOINT: "ws://devnet:8545"
+      CARTESI_BLOCKCHAIN_IS_LEGACY: "false"
+      CARTESI_BLOCKCHAIN_GENESIS_BLOCK: "1"
+      CARTESI_BLOCKCHAIN_READ_DEPTH: "1"
+      CARTESI_CONTRACTS_DAPP_ADDRESS: "0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C"
+      CARTESI_CONTRACTS_DAPP_DEPLOYMENT_BLOCK_NUMBER: "1"
+      CARTESI_CONTRACTS_HISTORY_ADDRESS: "0x4FF8BD9122b7D91d56Dd5c88FE6891Fb3c0b5281"
+      CARTESI_CONTRACTS_AUTHORITY_ADDRESS: "0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785"
+      CARTESI_CONTRACTS_INPUT_BOX_ADDRESS: "0x59b22D57D4f067708AB0c00552767405926dc768"
```
Comment (gligneul) [inline build/node-compose.yml:36]: @marcelstanley

**Review by torives — DISMISSED** [review]
It's so pretty it makes me wanna cry. And we're just warming up... 

Great work, guys 👏🏽

**Review by torives — APPROVED** [review]

**Review by renan061 — APPROVED** [review]

**Pull request merged**